berkeley-mobile/StudyPact/StudyPact.swift

```diff
@@ -94,6 +96,46 @@ class StudyPact {
             }
         }
     }
+    
+    // MARK: GetUser
+    
+    public func getUser(completion: @escaping (Bool) -> Void) {
+        guard let cryptohash = self.cryptoHash,
+              let email = self.email,
+              let url = EndpointKey.getUser.url else {
+            completion(false)
+            return
+        }
+        let params = ["user_email": email, "secret_token": cryptohash]
+        NetworkManager.shared.post(url: url, body: params, asType: AnyJSON.self) { response in
+            switch response {
+            case .success(_):
+                completion(true)
+            default:
+                completion(false)
+            }
+        }
+    }
```

Comment:
you'd want to create a decodable struct (like AuthenticateUserDocument) instead of using AnyJSON. and the completion block would then pass the profile data that is returned